triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -30,7 +34,11 @@ import Daml.Trigger.LowLevel hiding (Trigger)
 
 -- This will change to a Map once we have proper maps in DAML-LF
 data ACS = ACS
```

Comment:
Not for this PR but we could consider changing the list in the values to `Map AnyContractId AnyTemplate`. Then we can provide a fast lookup by contract id which is something you need fairly often.

Reply:
#7425